edb/server/dbview/dbview.pyx

```diff
@@ -1215,7 +1215,12 @@ cdef class DatabaseConnectionView:
             self.get_system_config(),
         )
 
-    async def recompile_cached_queries(self, user_schema, schema_version):
+    async def recompile_cached_queries(
+        self,
+        user_schema,
+        schema_version,
+        send_log_message: object | None = None,
```

Comment:
Ideally, this type annotation should say that it expects a function. Unless `.pyx` does not support that.